### PR TITLE
Added Sonarr, qBittorrent, Homarr, Jellyfin, Bazarr and Service-Template

### PR DESCRIPTION
**Pull request by jackspiering**
Added

### CODE REVIEW

File: services/qbittorrent/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# qBittorrent with Tailscale Sidecar Configuration
+
+This Docker Compose configuration sets up [qBittorrent](https://www.qbittorrent.org/) with Tailscale as a sidecar container to securely manage and access your torrent client over a private Tailscale network. By using Tailscale in a sidecar configuration, you can enhance the security and privacy of your qBittorrent instance, ensuring that it is only accessible within your Tailscale network.
+
+## Qbittorrent
```
Comment (crypt0rr) [inline services/qbittorrent/README.md:5]: Please rename `Qbitorrent` to `qBittorrent`

File: services/qbittorrent/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+services:
+# Make sure you replace SERVICE and SERVICEPORT. Environmentvariables may differ, check documentation of container maintaner.
```
Comment (crypt0rr) [inline services/qbittorrent/docker-compose.yml:2]: What does this mean? / What is the intention to have this here?

File: services/qbittorrent/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  # qbittorrent
+  qbittorrent:
+    image: lscr.io/linuxserver/qbittorrent:latest # Image to be used
+    network_mode: service:tailscale-qbittorrent # Sidecar configuration to route qbittorrent through Tailscale
+    container_name: qbittorrent # Name for local container management
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/Amsterdam
+      - WEBUI_PORT=8080 #@crypt0rr, I'm not sure if this is useful here. Remove if you disagree.
```
Comment (crypt0rr) [inline services/qbittorrent/docker-compose.yml:42]: Looks to be necessary to let the machine bind on 8080. You can remove the comment in the config file before we merge.

File: services/sonarr/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
+services:
+# Make sure you replace SERVICE and SERVICEPORT. Environmentvariables may differ, check documentation of container maintaner.
```
Comment (crypt0rr) [inline services/sonarr/docker-compose.yml:2]: What does this mean? / What is the intention to have this here?

File: templates/service-template/docker-compose.yml
Comment (crypt0rr) [inline templates/service-template/docker-compose.yml:1]: Something goes wrong here, looks like a replacement has been done that replaced current values for placeholders. Please fix.

**Review by crypt0rr — CHANGES_REQUESTED** [review]
Please fix the comments, added per file.

File: services/bazarr/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# SERVICE with Tailscale Sidecar Configuration
```
[the remark this answers is not in the document]
Comment (jackspiering) [reply services/bazarr/README.md:1]: Fixed

File: services/bazarr/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,54 @@
+services:
+# Make sure you replace SERVICE and SERVICEPORT. Environmentvariables may differ, check documentation of container maintaner.
```
[the remark this answers is not in the document]
Comment (jackspiering) [reply services/bazarr/docker-compose.yml:2]: Fixed, I forgot to delete it from the template

File: services/homarr/docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,51 @@
+services:
+# Make sure you replace SERVICE and SERVICEPORT. Environmentvariables may differ, check documentation of container maintaner.
```
[the remark this answers is not in the document]
Comment (jackspiering) [reply services/homarr/docker-compose.yml:2]: Fixed

File: services/jellyfin/docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,59 @@
+services:
+# Make sure you replace SERVICE and SERVICEPORT. Environmentvariables may differ, check documentation of container maintaner.
```
[the remark this answers is not in the document]
Comment (jackspiering) [reply services/jellyfin/docker-compose.yml:2]: Fixed

File: services/plex/README.md
[the remark this answers is not in the document]
Comment (jackspiering) [reply services/plex/README.md:1]: No idea, I made a mistake.

File: services/qbittorrent/README.md (hunk 1 above)
Comment (jackspiering) [reply services/qbittorrent/README.md:5]: Fixed

File: services/qbittorrent/docker-compose.yml (hunk 3 above)
Comment (jackspiering) [reply services/qbittorrent/docker-compose.yml:42]: Done

File: services/sonarr/docker-compose.yml (hunk 4 above)
Comment (jackspiering) [reply services/sonarr/docker-compose.yml:2]: Fixed

File: services/qbittorrent/docker-compose.yml (hunk 2 above)
Comment (jackspiering) [reply services/qbittorrent/docker-compose.yml:2]: Fixed

File: templates/service-template/docker-compose.yml
Comment (jackspiering) [reply templates/service-template/docker-compose.yml:1]: Added the original plex structure again, but changed the media and config paths.

File: services/qbittorrent/docker-compose.yml (hunk 2 above)
Comment (crypt0rr) [inline services/qbittorrent/docker-compose.yml:2]: The template text is still here, please remove

**Review by crypt0rr — CHANGES_REQUESTED** [review]
1 little change required

File: services/qbittorrent/docker-compose.yml (hunk 2 above)
Comment (jackspiering) [reply services/qbittorrent/docker-compose.yml:2]: Fixed

**Review by crypt0rr — APPROVED** [review]
Mighty, not tiny 🎉

**Pull request merged**